.ddev/config.local.yaml.example

```diff
@@ -54,11 +54,17 @@ hooks:
     - exec: "cd .. && rm -f hedley/modules/custom/hedley_migrate/csv/person.csv"
     - exec: "cd .. && rm -f hedley/modules/custom/hedley_migrate/csv/village.csv"
     # Enabling features:
+    - exec: drush vset hedley_admin_feature_acute_illness_enabled 1
```

Comment:
Does it make sense to add it to some kind of update hook too, even selectively for non-live environments?

Reply:
Great idea. Done.